cmd/postgres.go

```diff
@@ -675,9 +676,9 @@ func runListPostgresDatabases(cmdCtx *cmdctx.CmdContext) error {
 		return fmt.Errorf("get app: %w", err)
 	}
 
-	if !isPostgresApp(&app.ImageDetails) {
-		return fmt.Errorf("%s is not a postgres app", cmdCtx.AppName)
-	}
+	// if !isPostgresApp(&app.ImageDetails) {
```

Comment:
Are we checking this somewhere else?

Reply:
@davissp14 I only removed this temporary for testing, cause pg apps deployed with `fly deploy` fail to pass this checks